Homework7/FightLogic.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Linq;
+
+namespace Homework7
+{
+    public class FightLogic
```

Comment:
Still don't like this class.

Reply:
moved logic creating units/groups in Main, in